src/MICore/CommandFactories/lldb.cs

```diff
@@ -46,6 +46,25 @@ public override bool SupportsFrameFormatting
             }
         }
 
+        protected override StringBuilder BuildBreakInsert(string condition)
+        {
+            // LLDB's use of the pending flag requires an optional parameter or else it fails.
+            // We will use "on" for now. 
+            // TODO: Fix this on LLDB-MI's side
+            string pendingFlag = "-f on ";
```

Comment:
I wanted to call out specifically what was different and why it was different. I'll keep it separate for now until we fix LLDB side